website_and_docs/content/documentation/support_packages/chrome_devtools.en.md

```diff
@@ -5,16 +5,21 @@ weight: 5
 aliases: ["/documentation/en/support_packages/chrome_devtools/"]
 ---
 
-Selenium 4 alpha versions have much awaited native support for Chrome DevTools 
-Protocol through "DevTools" interface. This helps us getting Chrome Development 
-properties such as Application Cache, Fetch, Network, Performance, Profiler, 
-Resource Timing, Security and Target CDP domains etc.
-
-Chrome DevTools is a set of web developer tools built directly into the Google 
-Chrome browser. DevTools can help you edit pages on-the-fly and diagnose 
-problems quickly, which ultimately helps you build better websites, faster.
-
-## Emulate Geo Location:
+{{% pageinfo color="warning" %}}
+<p class="lead">
+  While Selenium 4 provides direct access to Chrome DevTools Protocol, it is highly encouraged that
+you use the <a href="../../webdriver/bidi_apis">WebDriver Bidi APIs</a> instead.
+</p>
+{{% /pageinfo %}}
+
+Chrome DevTools is a set of web developer tools built directly into the Google
+Chrome browser. The implementation is not designed for testing, nor to have a stable API,
+so functionality is highly dependent on the version of the browser.
+If there is DevTools functionality that is not yet covered by a Bidi API, though, you can
+use DevTools directly to access properties such as Application Cache, Fetch, Network, Performance, Profiler, 
```

Comment:
The examples are a hostage to the future: as bidi evolves, this will change. Maybe something like:

"If there is DevTools functionality that is not yet covered by a Bidi API, you can use the CDP directly to access that functionality"

Reply:
I like it, and I'll reference the browser version limitation again:
> If there is DevTools functionality that is not yet covered by a Bidi API, though, you can
use the CDP directly to access that functionality. Here are a couple examples that work as of
Chrome v93.